### DISCUSSION TITLE
Custom User Model For Achievements

### DISCUSSION THREAD

**Discussion by matthewscalf** [cjmellor/level-up#11] [Ideas]
When utilizing this package, the Achievements are cool idea but the migrations and Pivot models assume you have named the User model users. But you have the option to customize that for the other functions. Would be good to see the achievements utilize the same level of customization.

**Comment by cjmellor** [comment]
I think my initial thinking was you'd only ever want to give this to a User model, but it's a good point to have it customisable throughout. I'll look into it

**Comment by matthewscalf** [comment]
Awesome. In our app, we named it Customer, lol. Sounded good at the time.

**Comment by cjmellor** [comment]
@matthewscalf Can you check #13 and see if this is along the right tracks of what you're discussing? Feel free to amend further or point me in some direction.

Thanks

**Reply by matthewscalf** [reply]
Yeah that is looking great!

**Reply by cjmellor** [reply]
You can get the updates with the latest release 0.0.3

**Reply by matthewscalf** [reply]
Nice, worked great so far! Thanks for considering the feature.
